### PR TITLE
14895 Destination MongoDB Strict Encrypt Fixed Failing Integration Test

### LINKED ISSUE #14895: Destination MongoDB Strict Encrypt Failing Integration Test
## Environment
- **Airbyte version**:  example is 0.22.0-alpha
- **OS Version / Instance**: N/A
- **Deployment**: N/A
- **Source Connector and version**: N/A 
- **Destination Connector and version**: MongoDB Strict Encrypt 
- **Step where error happened**: Integration test fails 

## Current Behavior
Process completed with exit code 1.

## Expected Behavior
Zero exit code 

## Logs

`> Task :airbyte-integrations:bases:standard-destination-test:compileJava FAILED
/actions-runner/_work/airbyte/airbyte/airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java:1043: error: incompatible types: ConnectorJobOutput cannot be converted to ConnectorSpecification
            .run(new JobGetSpecConfig().withDockerImage(getImageName()), jobRoot);`
            
https://github.com/airbytehq/airbyte/runs/7419226840?check_suite_focus=true

## Steps to Reproduce
1. Go to [integration-test](https://github.com/airbytehq/airbyte/actions/workflows/test-command.yml) within Github actions tab 
2. On the right hand side, click `Run workflow`
3. Within the `Airbyte Connector` field type in `destination-mongodb-strict-encrypt`
4. Click the green `Run workflow` button to execute the integration test, be sure to note the test (#id_number) to verify the failure later

## Are you willing to submit a PR?

No, will assign this to GL to review and see if there's a particular reason the failures exist

### PR DESCRIPTION
**Pull request by suhomud**
## What
Destination MongoDB Strict Encrypt integration tests were failing because of server availability.  

## How
Integration tests were adapted to use mongo with test container

### CODE REVIEW

File: airbyte-integrations/connectors/destination-mongodb-strict-encrypt/src/test-integration/java/io/airbyte/integrations/destination/mongodb/MongodbDestinationStrictEncryptAcceptanceTest.java [hunk 1]
```diff
@@ -104,23 +127,27 @@ protected List<JsonNode> retrieveRecords(final TestDestinationEnv testEnv,
...
 
   @Override
-  protected void setup(final TestDestinationEnv testEnv) {
-    final String connectionString = String.format("mongodb://%s:%s@%s:%s/%s?authSource=admin&ssl=true",
+  protected void tearDown(final TestDestinationEnv testEnv) throws Exception {
+    container.close();
+  }
+
+  private MongoDatabase getMongoDatabase() {
+    final String connectionString = String.format("mongodb://%s:%s@%s:%s/%s?authSource=admin&tcl=true",
```
Comment (alexandr-shegeda) [inline airbyte-integrations/connectors/destination-mongodb-strict-encrypt/src/test-integration/java/io/airbyte/integrations/destination/mongodb/MongodbDestinationStrictEncryptAcceptanceTest.java:135]: please replace `tcl` with `tls`

File: airbyte-integrations/connectors/destination-mongodb-strict-encrypt/src/test-integration/resources/mongodb-cert.crt [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+9q6DmKKkVjqM1fbV9WK45tKzAZ3Quy1JXCT1BfLhkVXK3B0LsuX2JnmPVp7O5cAF
+aj/7hyAMmvI9ZnZWIB5LleiSTLI3QTAyv+j1rYZtPo7M4gceGRKLHIDKu8wL+kEt
+j8sZJTxifKeFSjPwBYwbgZNnHSUCAwEAATANBgkqhkiG9w0BAQsFAAOCAQEAFzDl
+pxrcHJFUgf4rNXFvND+iq4K1AtqEnd4Ze7N3pSHEIUj4YNdiMZP06nP5FAJklRkP
+GsMMeR8s+jfZtPgTlwrW2IKcdYZAkA91VoEDs6IK2VzC7GiEsUPaGh4/2/DjXkSZ
+9Ie4a4zfRVHCvkRMO5q9/KKpdTo1vwc83mmxMSKQKEv8NKbEhVc3VlCZV1MvpMdc
+7He0Jqfg53+yLZv33w6NA4X8QIWdoBuK02ZP7ZhEBcMJOByetsuaHXjadsOpP4LM
+7KaP6zUq9kteKD6TgiCTmV6U/tpDvhS3YdVBEdd0aPlBeWGJ+GzubbOUUJOYSkt0
+jQ/Z56qy7eSveCa5ZQ==
+-----END CERTIFICATE-----
```
Comment (andriikorotkov) [inline airbyte-integrations/connectors/destination-mongodb-strict-encrypt/src/test-integration/resources/mongodb-cert.crt:20]: What is the expiration date of these keys? We need them to be valid for a very long time.

**Review by andriikorotkov — APPROVED** [review]

**Pull request closed without merging**